src/components/AppNavigation/CalendarList/CalendarListItem.vue

```diff
@@ -488,3 +488,11 @@ export default {
 	},
 }
 </script>
+<style lang="scss" scoped>
+::v-deep .action-input {
+  margin-top: 0 !important;
+}
+::v-deep .action-button {
+  width: 217px !important;
```

Comment:
This seems to be a very specific width. Without checking, I guess this might fail for other content!?

Reply:
This with is only applied to calendarlistitem, so only for the "action menu" and i dindt find any think breaking on that. I am fine with other ideas how to fix it if you have any. This worked, and i pushed, im pretty new with calendar code :D